source/core/security-users.txt

```diff
@@ -145,6 +145,14 @@ the first user in the ``admin`` database. The first user must have
 privileges to create other users, such as a user with the
 :authrole:`userAdmin` or :authrole:`userAdminAnyDatabase` role.
 
+.. versionchanged:: 3.4
+
+   The localhost exception was extended to include the execution of the
+   :method:`db.createRole()` method. This provides users authorizing via LDAP
```

Comment:
This phrasing could be slightly tighter. Consider `This method allows users authorizing via LDAP to create...`